components/StyledAmountPicker.js

```diff
@@ -110,57 +97,6 @@ const StyledAmountPicker = ({ presets, currency, value, otherAmountDisplay, onCh
           }}
         </StyledButtonSet>
       )}
-      {otherAmountDisplay === 'input' && (
```

Comment:
This was dead code from the old way we used to deal with custom contribution amounts.